spec/rule_spec.rb

```diff
@@ -30,6 +30,80 @@
     end
   end
 
+  describe "iterating with a floor_date" do
```

Comment:
nit: mind using single quotes for consistency here?

Reply:
Done! 
https://github.com/square/ruby-rrule/pull/11/commits/4cdc9d97d07fe56f9a8cd85604682ca4327fdfb8